cni/cmd/tc-redirect-tap/main.go

```diff
@@ -15,7 +15,10 @@ package main
 
 import (
 	"encoding/json"
+	args2 "github.com/firecracker-microvm/firecracker-go-sdk/cni/cmd/tc-redirect-tap/args"
```

Comment:
nit: i'd prefer we rename this import to something like "pluginargs" which is a little more descriptive